pyproject.toml

```diff
@@ -85,12 +86,8 @@ dev = [
   "anthropic",
   "prometheus_client",
 ]
-evals = [
-  "arize-phoenix-evals>=0.3.0",
-]
-experimental = [
-  "tenacity",
```

Comment:
Did we forget to remove this previously?

Reply:
yeah, I think so